content/posts/blog/first-krakow-xmpp-meetup.md

```diff
@@ -0,0 +1,20 @@
+---
+title: First Krakow XMPP meetup
+date: 2017-04-28 12:59
+author: Nÿco
+blog_id: blog
+category: Events
+---
+
+Following and reinforcing the trend of XMPP meetups across Europe (Berlin, Stockholm, London, and Paris), here is Kraków!
+
+Tigase and MongooseIM teams will join forces to offer you three presentations:
+* "Why use XMPP for IoT?" by Andrzej Wojcik from Tigase
+* "ICE, STUN, and TURN" by Rafal Slota and Szymon Mentel from Erlang Solutions
+* "Fantastic XMPP use-cases" by Piotr Nosek from Erlang Solutions
+
+It will take place on the 10th of May, it is a Wednesday.
+
+All details available here: https://www.meetup.com/XMPP-Jabber-Messaging-Krakow/events/239508291/
+
+Please [join the "XMPP/Jabber Messaging Kraków" group](https://www.meetup.com/XMPP-Jabber-Messaging-Krakow/) and [RSVP](https://www.meetup.com/XMPP-Jabber-Messaging-Krakow/events/239508291/).
```

Comment:
I would combine the last two lines, as they both relate to the meetup.com page. Also, I'd hide the ugly URL. behind a descriptive lable. Something like this perhaps:

`All details are available at [meetup.com](https://www.meetup.com/XMPP-Jabber-Messaging-Krakow/events/239508291/)! Please [join the "XMPP/Jabber Messaging Kraków" group](https://www.meetup.com/XMPP-Jabber-Messaging-Krakow/) and [RSVP](https://www.meetup.com/XMPP-Jabber-Messaging-Krakow/events/239508291/).`